query-connector/src/app/utils/configChecks.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Checks if the property DEMO_MODE is true (Configured this way in demo env only)
+ * @returns true if the application is running in demo env
+ */
+export function isDemoMode(): boolean {
+  return process.env.DEMO_MODE === "true";
+}
+
+/**
+ * Checks if the property AUTH_DISABLED is true
+ * @returns true if auth is disabled for the application
+ */
+export function isAuthDisabled(): boolean {
+  return process.env.AUTH_DISABLED === "true";
+}
```

Comment:
I am not convinced the `configChecks.ts` is the right name for this file.

Reply:
could we just stick these two methods in `auth.ts`?

Reply:
This gave me an idea. I created an auth.ts file in the utils folder!